scriptmodules/emulators/hypseus.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hypseus"
+rp_module_desc="Hypseus Singe - Laserdisc Emulator"
+rp_module_help="ROM Extension: .daphne\n\nCopy your Daphne roms to $romdir/daphne"
+rp_module_licence="GPL3 https://raw.githubusercontent.com/DirtBagXon/hypseus-singe/master/LICENSE"
+rp_module_repo="git https://github.com/DirtBagXon/hypseus-singe.git RetroPie"
+rp_module_section="exp"
+rp_module_flags="sdl2"
+
+function depends_hypseus() {
+    getDepends libvorbis-dev libogg-dev zlib1g-dev libmpeg2-4-dev libsdl2-image-dev libsdl2-ttf-dev cmake
+}
+
+function sources_hypseus() {
+    gitPullOrClone "$md_build" "$md_repo_url" "$md_repo_branch" "" 0
+}
+
+function build_hypseus() {
+    local params=()
+    mkdir build
+    cd build
+    cmake ../src
+    make
+    cp hypseus ../hypseus.bin
+    md_ret_require="hypseus"
+}
+
+function install_hypseus() {
+    md_ret_files=(
+        'sound'
+        'pics'
+        'fonts'
+        'hypseus.bin'
+        'LICENSE'
+    )
+}
+
+function configure_hypseus() {
+    mkRomDir "daphne"
+    mkRomDir "daphne/roms"
+
+    addEmulator 1 "$md_id" "hypseus" "$md_inst/hypseus.sh %ROM%"
```

Comment:
This should be changed to
```
addEmulator 0 "$md_id" "daphne" "$md_inst/hypseus.sh %ROM%"
```
to ensure that `hypseus` is added as an alternative (and non-default) emulator for the `daphne` system.